docker-micro-benchmark/helpers/docker_helpers.go

```diff
@@ -244,9 +285,9 @@ func DoParallelContainerStopBenchmark(client *docker.Client, qps float64, routin
 
 // GetContainerIDs returns all the container ids in the system
 func GetContainerIDs(client *docker.Client) (containerIDs []string) {
-	containers, err := client.ListContainers(docker.ListContainersOptions{All: true})
+	containers, err := client.ContainerList(getContext(), types.ContainerListOptions{All: true})
 	if err != nil {
-		panic(fmt.Sprintf("Error list containers: %v", err))
+		LogError("Error list containers: %v", err)
```

Comment:
A more general question is does the tests retry the failed requests, or just continue, and why.